framework/web/Request.php

```diff
@@ -203,17 +212,31 @@ class Request extends \yii\base\Request
         'X-Forwarded-Proto',
         'Front-End-Https',
     ];
-
     /**
-     * @see $trustedHostConfig
-     * @var string[] List of headers where proxies store the real client IP, it's not advisable to put insecure headers
-     * here.
+     * @var string[] List of headers where proxies store the real client IP.
+     * It's not advisable to put insecure headers here.
+     * The match of header names is case-insensitive.
+     * @see $trustedHosts
+     * @see $secureHeaders
      * @since 2.0.13
      */
     public $ipHeaders = [
         'X-Forwarded-For'
     ];
     /**
+     * @var array list of headers to check for determining whether the connection is made via HTTPS.
+     * The array keys are header names and the array value is a list of header values that indicate a secure connection.
+     * The match of header names and values is case-insensitive.
+     * It's not advisable to put insecure headers here.
+     * @see $trustedHosts
+     * @see $secureHeaders
+     * @since 2.0.13
+     */
+    public $secureProtocolHeaders = [
```

Comment:
This naming is inconsistent with `$ipHeaders`.
Possibly rename `$ipHeaders` to `$secureIpHeaders` for consistency?